Galaxy.html

```diff
@@ -0,0 +1,67 @@
+					<li><input type="radio" name="planet" value="2"><i>Mars</i></li>
+					<li><input type="radio" name="planet" value="3"><i>Io</i></li>
+					<li><input type="radio" name="planet" value="4"><i>Phobos</i></li>
+					<li><input type="radio" name="planet" value="5"><i>GJ 273 b</i></li>
+					<li><p class="voting1"><input type="submit" value="Choose your destiny"></p></li>
+					<h3>«That's one small step for a man, one giant leap for mankind».(с)Neil Armstrong</h3>
+			</ul>	  
+			<div class="content">
+					<p><a href="https://en.wikipedia.org/wiki/Milky_Way"><h3>Milky Way</h3></a>
+					The Milky Way is the galaxy that contains our Solar System, with the name describing the galaxy's appearance from Earth: a hazy band of light seen in the night sky formed from stars that cannot be individually distinguished by the naked eye.The Milky Way is the galaxy that contains our Solar System, with the name describing the galaxy's appearance from Earth: a hazy band of light seen in the night sky formed from stars that cannot be individually distinguished by the naked eye.</br>
+					<img src="milkyway.jpg" alt="milky way"/>
+					</p>
+					<p><a href="https://en.wikipedia.org/wiki/Black_hole"><h3>Black hole</h3></a>
+					A black hole is a region of spacetime where gravity is so strong that nothing—no particles or even electromagnetic radiation such as light—can escape from it. The theory of general relativity predicts that a sufficiently compact mass can deform spacetime to form a black hole. The boundary of the region from which no escape is possible is called the event horizon.</br>
+					<img src="blackhole.jpg" alt="black hole"/>
+					</p>
+					<p><a href="https://en.wikipedia.org/wiki/Nebula"><h3>Nebula</h3></a>
+					A nebula (Latin for 'cloud' or 'fog'; pl. nebulae, nebulæ or nebulas) is an interstellar cloud of dust, hydrogen, helium and other ionized gases. Originally, the term was used to describe any diffused astronomical object, including galaxies beyond the Milky Way.</br>
+					<img src="nebula.jpg" alt="nebula"/>
+					</p>
+					<p><a href="https://en.wikipedia.org/wiki/Sun"><h3>Sun</h3></a>
+					The Sun is the star at the center of the Solar System. It is a nearly perfect sphere of hot plasma, with internal convective motion that generates a magnetic field via a dynamo process. It is by far the most important source of energy for life on Earth. Its diameter is about 1.39 million kilometers (864,000 miles), or 109 times that of Earth, and its mass is about 330,000 times that of Earth. It accounts for about 99.86% of the total mass of the Solar System.</br>
+					<img src="sun.jpg" alt="sun"/>
+					</p>		
+			</div>
+
+		</div>
+		
+		<footer class="footer">
+			<nav class="other">
+					<a href="#">Blog</a>
+					<a href="#">Events</a>
+					<a href="#">Partners</a>
+					<a href="#">Legal</a>
+			</nav>	
+			<div class="social">
+					<a href="#"><img src="tw.png" alt="tw"/></a>
```

Comment:
Put these social icons as you have done but on one line with group of links above, f.e. in proportion 60/40 %:
60 - nav links, 40 - social links(if you have any social account - add real link please, so it will be opened in separate tab)

Reply:
> Put these social icons as you have done but on one line with group of links above, f.e. in proportion 60/40 %:
> 60 - nav links, 40 - social links(if you have any social account - add real link please, so it will be opened in separate tab)

design and link in social account will be changed